projects/wipp-frontend-lib/src/lib/notebook/notebook-detail/notebook-detail.component.ts

```diff
@@ -21,6 +22,8 @@ import {HttpClient} from '@angular/common/http';
 import {NgxSpinnerService} from 'ngx-spinner';
 import {ModalErrorComponent} from '../../modal-error/modal-error.component';
 import {PlatformLocation} from '@angular/common';
+declare var nb: any;
```

Comment:
Can you add a comment describing why this declaration is needed?

Reply:
the nb variable used to be decalred in a typing.d.ts file along with the WDZT variable. When trying to use the typing.d.ts, both variables were causing not found errors. The only way I was able to make it work was by declaring both variables inside the components. I will add a comment explaining it.